### DISCUSSION TITLE
Implement a transient webpage sidebar feature (similar to Firefox Side View)

### DISCUSSION THREAD

**Discussion by win8linux** [Floorp-Projects/Floorp#647] [Feature Request]
As an alternative to fixing https://github.com/Floorp-Projects/Floorp/issues/646, the functionality of [Firefox Side View](https://github.com/mozilla/side-view/) extension could be reimplemented into Floorp itself instead. While it is already possible to create a Web Panel, this is more permanent and takes more steps than what Firefox Side View provides.

**Comment by CutterKnife** [comment]
Thank you for your idea.
One immediate means is the Split View feature. Right-click on the tab and select it from the menu.

**Reply by win8linux** [reply]
This is close, however it still requires that a tab be present and it is not resizable. Perhaps it could be expanded or changed to work closer to Mozilla's FF Side View, but better integrated into Floorp's sidebar?
